sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizer/BindList.java

```diff
@@ -113,6 +74,6 @@ public SqlStatementParameterCustomizer createForParameter(Annotation annotation,
          */
         THROW;
 
-        static final String valueNotHandledMessage = "EmptyHandling type on BindList not handled. Please report this to the jdbi developers.";
+        public static final String valueNotHandledMessage = "EmptyHandling type on BindList not handled. Please report this to the jdbi developers.";
```

Comment:
This message isn't really part of the API. I think I'd rather move it to an internal package than expose it.